src/permissionGroups/views/PermissionGroupList/PermissionGroupList.tsx

```diff
@@ -83,7 +85,15 @@ export const PermissionGroupList: React.FC<PermissionGroupListProps> = ({
         })
       });
       refetch();
+      setDeleteError("");
```

Comment:
Prop is either string or undefined, but you use empty string here. This often causes subtle bugs like displaying a component with padding with no content and so on